thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/spi/model/AnomalySlice.java

```diff
@@ -62,28 +83,40 @@ public long getEnd() {
     return end;
   }
 
+  public boolean getIsTaggedAsChild() {
+    return isTaggedAsChild;
+  }
+
   public Multimap<String, String> getFilters() {
     return filters;
   }
 
+  public List<String> getDetectionCompNames() {
```

Comment:
minor nitpick, but is there a reason we are using DetectionCompNames rather than the full DetectionComponentNames?

Reply:
it shouldn't really matter as long as it is not conflicting or unclear.